doc/index.rst

```diff
@@ -669,6 +670,34 @@ Since v0.6.0, you can run locally using `.travis.yml` you already defined for yo
 
    rosrun industrial_ci run_travis --help
 
+Recurring runs for debugging
+++++++++++++++++++++++++++++
+Please note that `run_ci` and `run_travis` will download all dependencies every time, just as CI services would do.
+For recurring runs, e.g. in a debugging session, this might not be desired.
+
+As an alternative `rerun_ci` could be used. It take the same argument as `run_ci`, but will run the build incrementally and only download or compile after changes.
+This results in much faster execution for recurring runs, but has some disadvantages as well:
+* The user needs to clean-up manually, an instruction to do so is printed at the end of all runs.
+* All parameters incl. the repository path have to be passed explicitly to allow for proper caching.
+* The apt dependencies won't get updated in recurring runs.
+* Incremental builds might not work properly for all cases. Especially, it does not help with prerelease tests.
+
+Example:
+
+::
+
+  $ rosrun industrial_ci rerun_ci . ROS_DISTRO=melodic ROS_REPO=ros-shadow-fixed
+
+This will run the tests and commit the result to a Docker image `industrial-ci/rerun_ci/ros_canopen:$HASH`.
+The hash is unique for each argument list, so `rerun_ci . ROS_DISTRO=melodic` and `rerun_ci . ROS_DISTRO=kinetic` do not mix  up.
```

Comment:
> so `rerun_ci . ROS_DISTRO=melodic` and `rerun_ci . ROS_DISTRO=kinetic` do not mix  up.

github.com does not render it nicely enough so that it's hard to tell apart the 2 sets of commands. Maybe separate lines makes it the most clear.

Reply:
Apparently, inline code has to be wrapped in double backticks.